CHANGES.rst

```diff
@@ -157,6 +157,8 @@ Specviz2d
 Other Changes and Additions
 ---------------------------
 
+- Bump required specutils version to 1.16. [#3184]
```

Comment:
didn't this actually fix a bug in moment maps?  Should we spell that out as a bugfix in case anyone has that problem on 3.10.x?  And for that matter - should we backport or no?  cc @rosteen

Reply:
Typically a package does not backport a version bump unless it is a security patch.

Reply:
is it still worth giving more context in the changelog or does that not matter since the bug was fixed upstream?

Reply:
More info does not hurt.

Reply:
@cshanahan1 - up to you (if you remember the bug it fixed), otherwise feel free to merge now that tests are passing!

Reply:
if you're ok with the updated log entry ill enable squash and merge